### PR TITLE
[cmake] Update Effects11 and Build it with CMake instead VS

### PR DESCRIPTION
**Pull request by fetzerch**
ping: @Paxxi, @afedchin

### CODE REVIEW

File: xbmc/filesystem/CMakeLists.txt [hunk 1]
```diff
@@ -180,7 +180,7 @@ if(BLURAY_FOUND)
                       BlurayFile.h)
 endif()
 
-core_add_library(filesystem upnp)
+core_add_library(filesystem)
```
Comment (MartijnKaijser) [inline xbmc/filesystem/CMakeLists.txt]: Seems not related?

**Comment by afedchin** [comment]
did you update fx11 from upstream?

**Comment by Paxxi** [comment]
@afedchin it's updated with the latest upstream release from August 2

**Comment by afedchin** [comment]
oki. that's fine for me. thanks

File: xbmc/filesystem/CMakeLists.txt (hunk 1 above)
Comment (fetzerch) [inline xbmc/filesystem/CMakeLists.txt]: moved to separate commit

**Comment by fetzerch** [comment]
@Paxxi: I think it would be still good to get this in, even if we replace it with a prebuilt version from depends at some point.

With the changes in https://github.com/xbmc/xbmc/pull/10256 appveyor fails on the effects11 solution: https://ci.appveyor.com/project/MartijnKaijser/xbmc/build/1.0.2827.

Jenkins build this please

**Comment by Paxxi** [comment]
agreed, I also wanted this in for the x64 work because it also got stuck on building this
